addons/delivery/models/sale_order.py

```diff
@@ -114,7 +114,8 @@ def _create_delivery_line(self, carrier, price_unit, price_unit_in_description=F
             values['name'] += _(' (Estimated Cost: %s )') % self._format_currency_amount(price_unit)
         else:
             values['price_unit'] = price_unit
-
+        if additionnal_info:
```

Comment:
if carrier.free_over and self._compute_amount_total_without_delivery() >= carrier.amount:

(so you can remove all additional info stuff in signature)